book/src/install.md

```diff
@@ -151,6 +155,16 @@ This command will create the `hx` executable and construct the tree-sitter
 grammars in the local `runtime` folder. To build the tree-sitter grammars requires
 a c++ compiler to be installed, for example `gcc-c++`.
 
+To test for an installed `gcc-c++` on your Unix distribution try:
+
+```sh
+$ g++ --version
+
+g++ (Debian 10.2.1-6) 10.2.1 20210110
+Copyright (C) 2020 Free Software Foundation, Inc.
+...
+```
+
```

Comment:
I don't think this is necessary to add. Compiling the tree-sitter grammars with clang works just as well and this makes gcc-c++ look required.